logbook/types.go

```diff
@@ -0,0 +1,85 @@
+package logbook
+
+import (
+	"github.com/qri-io/qri/logbook/oplog"
+)
+
+// AuthorLog is the top-level log representing the book author
+type AuthorLog struct {
+	l *oplog.Log
+}
```

Comment:
I'd prefer this be `UserLog` and be generalized to _any_ top level user logbook entry, then have a method `authorLog` that returns the author, which is a preconfigured instance of a `UserLog`.

Reply:
:+1: